### DISCUSSION TITLE
Welcome to luci-app-nauta Discussions!

### DISCUSSION THREAD

**Discussion by cuza** [cuza/luci-app-nauta#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jaigos** [comment]
Buen trabajo hermano, un abrazo y esperamos que sigas con tus proyectos
